reya_actions/actions/bridge_in.py

```diff
@@ -0,0 +1,93 @@
+from dataclasses import dataclass
+import json
+from web3 import Web3
+from web3.middleware import construct_sign_and_send_raw_middleware
+
+@dataclass
+class BridgeInParams:
+    amount: int
+    fee_limit: int
+
+f = open('reya_actions/abis/SocketVaultWithPayload.json')
+vault_abi = json.load(f)
+
+f = open('reya_actions/abis/Erc20.json')
+erc20_abi = json.load(f)
+
+def bridge_in_from_arbitrum(config: dict, params: BridgeInParams):
+    arbitrum_rpc_url = 'https://arb1.arbitrum.io/rpc'
```

Comment:
should we have this as arg?

Reply:
maybe with this default value

Reply:
yes, but this is for reference. Anyone can plug in and adjust on their usecase.